employees/templatetags/data_display_filters.py

```diff
@@ -10,3 +12,20 @@
 @register.filter
 def duration_field_to_string(data: timedelta) -> str:
     return timedelta_to_string(data)
+
+
+@register.filter
+def extract_year_and_month_from_url(url: str) -> List[str]:
```

Comment:
This looks a little bit fragile in terms of if url will be modified in `urls.py`. We could use regex for this, which, if possible, includes current url. But I think its optional, so if you think its worth @kbeker  I can create separate issue for this.

Reply:
@rwrzesien Sure, create issue, but add there that it is now with low priority, but worth to do :)

Reply:
Done https://github.com/Code-Poets/sheetstorm/issues/295